client/Packages/com.beamable.server/Editor/Usam/Editor/CodeService.cs

```diff
@@ -99,24 +98,11 @@ public async Promise Init()
 
 		public async Promise UpdateServicesVersions()
 		{
-			var version = new BeamVersionResults();
-			var versionCommand = _cli.Version().OnStreamVersionResults(result =>
-			{
-				version = result.data;
-			});
-			await versionCommand.Run().Error(LogExceptionVerbose);
-
-			if (string.IsNullOrEmpty(version?.version) || version.version.Contains("1.0.0"))
-			{
-				LogVerbose("Could not detect current version, skipping");
-				return;
-			}
-
-			var versions = _cli.ProjectVersion(new ProjectVersionArgs { requestedVersion = version?.version });
+			var nugetVersion = GetCurrentNugetVersion();
+			var versions = _cli.ProjectVersion(new ProjectVersionArgs { requestedVersion = nugetVersion });
 			versions.OnStreamProjectVersionCommandResult(result =>
 			{
-				_projectVersion = result.data.packageVersions[0];
-				LogVerbose($"Versions updated: {_projectVersion}");

```

Comment:
I believe this is going to conflict with Peters migration PR, might want to merge his PR first